src/utilities/gql.ts

```diff
@@ -1340,6 +1340,18 @@ const gql = {
     }
   `,
 
+  SUB_SIMULATION_DATASETS: `#graphql
+    subscription SubSimulationDatasetIds($planId: Int!) {
+      simulation(where: { plan_id: { _eq: $planId } }, order_by: { id: desc }) {
+        simulation_datasets: simulation_datasets(order_by: { id: desc }) {
```

Comment:
We can remove the `simulation_datasets:` alias since the node is called `simulation_datasets`, they are equivalent